src/modding/ModManager.cs

```diff
@@ -433,10 +634,17 @@ private void OnOpened()
         RefreshAvailableMods();
         RefreshEnabledMods();
 
+        if (ModLoader.Instance.GetModErrors().Count > 0)
+        {
+            RefreshModErrors();
```

Comment:
What does this do? The mod manager can't have caused problems when it is opened, so I find this pretty useless. Or is this related to the not clearing the errors on game startup? Which is a change I don't like as I said before.

Reply:
I already changed back the unclearing thing, but this is so I can capture any startup mod errors so that it can be displayed in the mod errror tab/

Reply:
I'll have to re-read the changes once the minor style / structure things are resolved...